recipes/r-downloader/meta.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: r-downloader
+  version: 0.0.4
+
```

Comment:
The version number seems wrong here.
Please also have a look at https://github.com/conda-forge/r-stringi-feedstock/blob/master/recipe/meta.yaml for how to use templates.